### PR TITLE
add Gitpod vs Codespaces landing page.

### PR DESCRIPTION
**Pull request by nisarhassan12**
Fixes gitpod-io/website#766

### CODE REVIEW

File: src/pages/gitpod-vs-codespaces.tsx [hunk 1]
```diff
@@ -0,0 +1,385 @@
...
+                  <h2>Gitpod</h2>
+                </th>
+                <th>
+                  <h2>Codespaces</h2>
+                </th>
+              </tr>
+            </thead>
+            <tbody className="comparison">
+              <tr>
+                <th>Pricing(Hosted)</th>
```
Comment (ChristinFrohne) [inline src/pages/gitpod-vs-codespaces.tsx:124]: add spacing

File: src/pages/gitpod-vs-codespaces.tsx [hunk 2]
```diff
@@ -0,0 +1,385 @@
...
+                <td>Free for 50h/month</td>
```
Comment (ChristinFrohne) [inline src/pages/gitpod-vs-codespaces.tsx:125]: Change to "Free for Open-Source"

File: src/pages/gitpod-vs-codespaces.tsx [hunk 3]
```diff
@@ -0,0 +1,385 @@
...
+                <th>VS Code Extensions</th>
+                <td>
+                  <Tick />
+                </td>
+                <td>
+                  <Tick />
+                </td>
+              </tr>
+              <tr>
+                <th>iPad support</th>
```
Comment (ChristinFrohne) [inline src/pages/gitpod-vs-codespaces.tsx:241]: Please use a capital "S" for "support"

**Review by ChristinFrohne — CHANGES_REQUESTED** [review]
Thanks, it already looks great. 

- [x] please have the footnotes below the gray box and smaller

- [ ] the tooltips should appear on click, so that you have a chance to click on "learn more". The font should be regular 

- [x] the tooltips should be placed in the middle of the text (currently it's slightly above)

- [x] the browser-extension-button should link directly to the Chrome or Firefox extension. Depending on if the user is currently using a Chrome or Firefox browser, he/she should be forwarded to the correlating extension. 

We are still discussing some changes about the graphic. I'll get back to you with that feedback shortly.

**Comment by jankeromnes** [comment]
Many thanks for the excellent work @nisarhassan12!

---

Screenshot 2020-09-30 at 14 41 16

@ChristinFrohne Random thought, what you do think about using `+` instead of `,`? E.g.:

> Experience Gitpod, the [fast](#) + [cost-effective](#) + [open-source](#) alternative to Codespaces.

---

Screenshot 2020-09-30 at 14 42 50

The widths in the graphic should be proportional to the values:

- [ ] Codespaces' "Loading" bar (5min) should be 5x larger than Gitpod's (1min) -- currently it seems to be only ~2.5x larger

- [ ] Codespaces' "Project Build" (10min) should be 2x larger than the "Loading" bar (5min) -- currently it also seems to be ~2.5x larger

---

Screenshot 2020-09-30 at 14 46 53

There is already the "Last verified" text below with a date. So, please remove "as of September 2020" in the "(16 vCPU / 60GB RAM as of September 2020)".

Also the new values should be:

> (16 vCPU / 64GB RAM)

Because we're upgrading the nodes.

---

Screenshot 2020-09-30 at 14 48 47

- [ ] Please reduce the vertical empty space between the column headers (e.g. "Gitpod") and the first element (e.g. "Free for 50h/month")

- [ ] Please left-justify the "Self-Host on X" texts (all the "Self-Host" words should be vertically alligned to the left, regardless of logo width -- also it should be "Self-Host" and not "Self Host")

---

Screenshot 2020-09-30 at 14 51 42

- [ ] Please correctly format the number (`,` instead of `.`), and use the new value "Used by 250,000+ Developers & Businesses"

- [ ] Please remove the Mozilla logo, and add the GitLab logo to the left, as in the mockup

**Comment by ChristinFrohne** [comment]
Great feedback, thanks @jankeromnes 

>Experience Gitpod, the fast + cost-effective + open-source alternative to Codespaces

I prefer it the way it is. The "-" in cost-effective might look like a minus sign. Also, I'd expect an equation.

**Comment by nisarhassan12** [comment]
> the browser-extension-button should link directly to the Chrome or Firefox extension. Depending on if the user is currently using a Chrome or Firefox browser, he/she should be forwarded to the correlating extension.

@ChristinFrohne Thanks for pointing this out it will hopefully make the UX a lot better. What to do in case where the browser does not have the extension available for it let say Safari, for instance, we have a lot fo users from there I think Maybe link to the Chrome extension? What do you say? Thanks

**Comment by ChristinFrohne** [comment]
Thanks for the adjustments. Here some more comments:

- [ ] please have the footnote smaller and with less spacing to the box above

- [ ] link "Sources: Gitpod Pricing, Codespaces Pricing." to each pricing page. (Codespaces: https://docs.github.com/en/free-pro-team@latest/github/developing-online-with-codespaces/about-billing-for-codespaces)

- [ ] tooltip: the hover works great now. Thanks! Please give the text inside the tooltip a bit more margin to the left and the edges of the box should be slightly round

- [ ] All boxes should have slightly round corner. I think we set it to 3px. please adjust that to the two boxes comparing Gitpod and Codespaces as well

- [ ] typo: "coming soon" 

- [ ] the disclaimer should also have a smaller font size (the same as the footnotes)

- [ ] switch the section "used by" and "install browser extension", so the CTA button appears a bit sooner

Mobile view:

- [ ] have the same width for both columns
Bildschirmfoto 2020-10-02 um 12 23 46

- [ ] adjust the spacing for the footnotes (and have them smaller) so that the main text is easier to read
Bildschirmfoto 2020-10-02 um 12 24 35

**Comment by nisarhassan12** [comment]
>  All boxes should have slightly round corner. I think we set it to 3px. please adjust that to the two boxes comparing Gitpod and Codespaces as well

Sorry, I gave this a try can we skip this for now? Is that too important?

**Comment by nisarhassan12** [comment]
@ChristinFrohne I have made some new changes and also updated the Graphics please have a look whenever you'll have time. Thanks

**Comment by ChristinFrohne** [comment]
Cool, thanks!

- [ ] the link in the sub headline "resource-efficient" is not working

- [ ] please make sure that the first graphic is aligned with the "log in" button. Currently it's a bit too wide

- [ ] first graphic: there should be no white space between the two vertical lines and the horizontal bars

- [ ] the "loading" bar should be exactly half the size of the "project build" bar

- [ ] the size of the footnotes are inconsistent. Please always use the smaller size (like underneath the comparison table) and they should be left aligned with the gray box (see mockup)

- [ ] can we have the link to GCP directly in the 1? It takes too much attention in the lonely footnote, I feel.

- [ ] have less spacing after the comparison table and the CTA

- [ ] have more spacing above the headline (and a smaller footnote), so that there is a clear hierarchy (mobile view)
Screenshot 2020-10-06 110904


after you addressed these comments, the PR can be merged, so I can share it with the new editors. Further changes we can address in a separate PR. Thanks!

**Comment by nisarhassan12** [comment]
>  first graphic: there should be no white space between the two vertical lines and the horizontal bars

@ChristinFrohne there is already no space or did you meant something else? 

image

**Comment by nisarhassan12** [comment]
>  the size of the footnotes are inconsistent. Please always use the smaller size (like underneath the comparison table) and they should be left-aligned with the grey box (see mockup)

I have made the font size smaller. FYI alll the footnotes had the same size earlier i.e the ones under the graphics and the one under the Comparison Table.

Regarding the alignment, I tried to get it as it is in the mockup but unable to do so. Should I retry or Is it ok the way it is ? Thanks.

**Comment by ChristinFrohne** [comment]
The changes look good, thanks! 

Last small adjustments:

- [ ] we simplified the first graphic again. Please adjust it accordingly. Sorry about the back and forth (I updated the mockup)

- [ ] remove the headline of the second graphic and add it to the footnotes (see mockup)

- [ ] there is still too little space between the graphic/footnotes and the headlines on mobile view. Please add more space 
Bildschirmfoto 2020-10-07 um 10 15 02


after this version is merged, it would be great if you could have a look at a better mobile view for the table. (see https://www.pipedrive.com/en/crm-comparison/copper-alternative for inspiration)

File: src/contents/gitpod-vs-codespaces.tsx [hunk 4]
```diff
@@ -0,0 +1,65 @@
...
+import { Link } from 'gatsby'
+
+export const features: FeatureCardProps[] = [
+  {
+    id: 'fast',
+    title: <strong>Loads 15x Faster</strong>,
+    text: (
+      <>
+        <p>
+          Gitpod removes long init and build times by continuously pre-building workspaces for your project, allowing you to start coding or
```
Comment (ChristinFrohne) [inline src/contents/gitpod-vs-codespaces.tsx:15]: The sentence is quite long and thereby difficult to read. Let's break it into two sentences 
"Gitpod removes long init and build times by continuously pre-building workspaces for your project. Thereby it allows you to start coding or debugging immediately, from any context, at any time. "

**Review by ChristinFrohne — APPROVED** [review]

**Pull request merged**